packages/api-request-builder/package.json

```diff
@@ -19,9 +19,15 @@
   "bugs": "https://github.com/commercetools/nodejs/issues",
   "license": "MIT",
   "author": "Nicola Molinari <nicola.molinari@commercetools.com> (https://github.com/emmenko)",
-  "main": "dist/api-request-builder.cjs.js",
-  "module": "dist/api-request-builder.es.js",
-  "browser": "dist/api-request-builder.umd.js",
+  "main": "dist/commercetools-api-request-builder.cjs.js",
+  "module": "dist/commercetools-api-request-builder.esm.js",
+  "umd:main": "dist/commercetools-api-request-builder.umd.min.js",
```

Comment:
Sorry, I may've missed submitting this with the review.

For some reason the `commercetools-api-request-builder.umd.min.js` is not generated for all packages

Reply:
UMD only exists for legacy browser bundles. Not for any CLI tool.

Reply:
I only checked couple of packages and assumed that was the case for all packages